boxsdk/auth/oauth2.py

```diff
@@ -166,7 +181,7 @@ def authenticate(self, auth_code):
             data['box_device_id'] = self._box_device_id
         if self._box_device_name:
             data['box_device_name'] = self._box_device_name
-        return self.send_token_request(data, access_token=None)
+        return self.send_token_request_and_store_tokens(data, access_token=None)
```

Comment:
Sorry, I know I suggested this rename, but then I realized it's an unnecessary breaking change. Can you rename this back, and call the new method something else like `_send_token_request_without_storing_tokens`?

Reply:
+1

Reply:
Done